go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -1280,6 +1281,10 @@ func initSvcViaMgmPortRoutingRules(hostSubnets []*net.IPNet) error {
 		}
 	}
 
+	// lastly update the sysctl reverse path filtering options for ovn-k8s-mp0 interface to avoid dropping return packets
+	// NOTE: v6 doesn't have rp_filter strict mode block
+	_, _ = sysctl.Sysctl(fmt.Sprintf("net/ipv4/conf/%s/rp_filter", types.K8sMgmtIntfName), "2")
```

Comment:
shouldn't we at least log the error here and say ITP may not work correctly?

Reply:
maybe check the return value to, just to make sure it was set correctly?

Reply:
ack I guess we should actually treat this like any other error and return the error and fail for setup...let me clean this up tomorrow.

Reply:
done!